projects/adrv9371x/a10gx/system_project.tcl

```diff
@@ -133,4 +133,8 @@ set_instance_assignment -name IO_STANDARD "1.8 V" -to ad9371_gpio[16]
 set_instance_assignment -name IO_STANDARD "1.8 V" -to ad9371_gpio[17]
 set_instance_assignment -name IO_STANDARD "1.8 V" -to ad9371_gpio[18]
 
+## improve timing - there are ocassional timing failer with a small negative slack
```

Comment:
I think there is a typo " occasional timing failure" in both comments.

Reply:
Thanks! Fixed.